wiki/People/Users_with_unique_titles/id.md

```diff
@@ -89,45 +85,52 @@ Sebagian besar kontes mapping resmi menawarkan gelar **Elite Mapper** sebagai ha
 
-![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) menerima gelar ini setelah sebelumnya memiliki gelar Elite Mapper, jadi tim osu! meningkatkan levelnya menjadi **Elite Mapper II: Aspirant**. ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini lebih memilih untuk menggunakan gelar **osu!catch Champion**.
+¹ ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) saat ini menyandang gelar **Elite Mapper II: Aspirant** atas keberhasilannya dalam menjuarai [Monthly Beatmapping Contest #1](https://osu.ppy.sh/home/news/2013-10-20-monthly-beatmapping-contest-1-results) dan [Monthly Beatmapping Contest: Aspire (2014)](https://osu.ppy.sh/home/news/2014-11-13-monthly-beatmapping-contest-aspire-results) secara bersamaan.
+
+² Atas permintaan dari pihak yang bersangkutan, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini menyandang gelar **osu!catch Champion**.
 
-### Pemenang kontes osu!remix
+### Pemenang kontes-kontes osu!remix
 
-Kontes osu!remix menghadiahkan gelar **osu!mixer** sebagai hadiah utama dari kontes tersebut.
+osu! menganugerahkan gelar **osu!mixer** kepada para pemenang kontes-kontes osu!remix.
 
-- ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471)
+- ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471)¹
 - ![][flag_FR] [Shinwoir](https://osu.ppy.sh/users/8984574)
 
-### Artis unggulan
+¹ ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) saat ini menyandang gelar **osu!mixer II** atas keberhasilannya dalam menjuarai dua kontes osu!remix yang berbeda, yakni [osu!remix contest #1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) dan [A Labour of Love](wiki/en/Contests/A_Labour_of_Love_Contest).
+
+### Featured Artist
 
-[Artis unggulan](/wiki/Featured_Artists) yang memiliki akun osu! memiliki gelar **Featured Artist** untuk menunjukkan keterlibatan mereka dalam game ini.
+osu! menganugerahkan gelar **Featured Artist** kepada para [Featured Artist](/wiki/Featured_Artists) yang memiliki akun osu! sebagai salah satu bentuk balas jasa atas lagu-lagu yang telah mereka sediakan.
 
-![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) dan ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) memiliki gelar **osu!musician** atas kontribusinya di masa awal osu! dan merancang konsep musik dan tema resmi osu!, dan ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) memilih untuk menampilkan gelar **osu!mixer** miliknya ketika ia menjadi Artis Unggulan.
+![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) dan ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) saat ini menyandang gelar **osu!musician** atas kontribusi mereka dalam menata efek-efek suara dan menggubah lagu tema resmi yang dimiliki oleh osu!. Di sisi lain, ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) lebih memilih untuk mempertahankan gelar **osu!mixer** yang dimilikinya di saat ia diangkat menjadi Featured Artist.
```

Comment:
```suggestion
![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) dan ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) saat ini menyandang gelar **osu!musician** atas kontribusi mereka dalam menata efek-efek suara dan mengubah lagu tema resmi yang dimiliki oleh osu!. Di sisi lain, ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) lebih memilih untuk mempertahankan gelar **osu!mixer** yang dimilikinya di saat ia diangkat menjadi Featured Artist.
```

Reply:
oh, this is not a typo tho -- the is indeed meant to be `menggubah` instead of `mengubah`

- [Menggubah](http://kbbi.web.id/gubah) : "composing, (to) compose"
- Mengubah : "changing"